dmriprep/cli/run.py

```diff
@@ -221,16 +221,11 @@ def main():
 
     exec_env = os.name
 
-    # special variable set in the container
-    if os.getenv('IS_DOCKER_8395080871'):
-        exec_env = 'singularity'
-        cgroup = Path('/proc/1/cgroup')
-        if cgroup.exists() and 'docker' in cgroup.read_text():
-            exec_env = 'docker'
-            if os.getenv('DOCKER_VERSION_8395080871'):
-                exec_env = 'dmriprep-docker'
```

Comment:
I also removed this, because I *believe* this is a vesitigial remnant of sentry

Reply:
Indeed it is.